ui/src/app/workflow-templates/components/workflow-template-details/workflow-template-details.tsx

```diff
@@ -2,19 +2,19 @@ import {NotificationType, Page} from 'argo-ui';
 import {SlidingPanel} from 'argo-ui/src/index';
 import * as React from 'react';
 import {RouteComponentProps} from 'react-router';
-import {Workflow, WorkflowTemplate} from '../../../../models';
+import * as models from '../../../../models';
```

Comment:
minor - I'm not a fan of * import, not sure why - confusing maybe

Reply:
Not super opinionated, but can remove if necessary